application.py

```diff
@@ -3,7 +3,23 @@
 import os
 import re
 from app import create_app
-from flask.ext.script import Manager, Server
+from flask.ext.script import Manager, Server as _Server
+
+
+def extra_files():
+    for dirname, dirs, files in os.walk('./app/content'):
+        for filename in files:
+            filename = os.path.join(dirname, filename)
+            if os.path.isfile(filename):
+                print(filename)
+                yield filename
+
+
+class Server(_Server):
```

Comment:
I'd go with a different class name since we don't really need to pretend that it's the original `Server` class